addons/website_event_meet/tests/test_website_event_meet.py

```diff
@@ -29,6 +29,8 @@ def test_meeting_room_create(self):
         self.assertTrue(meeting_room.chat_room_id)
         self.assertTrue(meeting_room.chat_room_id.name)
         self.assertEqual(meeting_room.chat_room_id.max_capacity, "8")
+        # Ensure default value for room_max_capacity in event.meeting.room
+        self.assertEqual(meeting_room.room_max_capacity, "8")
```

Comment:
Could be great to check that ``max_capacity`` on linked ``chat.room`` is also 8

Reply:
Hello Sir, Line number 31 already checks max_capacity in linked chat.room is "8". Do i have to add that again?

Reply:
No it is ok, did not see it :)

Reply:
ok sir